prow/githuboauth/githuboauth.go

```diff
@@ -163,7 +194,7 @@ func (ga *Agent) HandleRedirect(client OAuthClient, getter GitHubClientGetter) h
 		}
 		secretState, ok := oauthSession.Values[stateKey].(string)
 		if !ok {
-			ga.serverError(w, "Get secret state", fmt.Errorf("empty string or cannot convert to string"))
+			ga.serverError(w, "Get secret state", fmt.Errorf("empty string or cannot convert to string: %v", oauthSession.Values))
```

Comment:
Can we make this error more clear? I think everyone that hits it when setting up OAuth has no idea what it means.

Reply:
Also, does this log sensitive data?